client/avatar.jsx

```diff
@@ -5,10 +5,17 @@ var Avatar = React.createClass({
         return 'https://avatars.githubusercontent.com/' + username.toLowerCase();
     },
     render() {
-        const src = this._getAvatar(this.props.username);
+        var ret = null;
+
+        if(this.props.username != null) {
+            var src = this._getAvatar(this.props.username);
+            ret = <img src={src} alt={this.props.username} className='avatar' onClick={this.props.toggleUserMenu} />;
+        }
 
         return (
-            <img src={src} alt={this.props.username} className='avatar' />
+            <div>
+                {ret}
```

Comment:
Yes, because Avatar react class is used in the topbar. So until the user login, the 'username' is empty. I need to return something each time so I return an empty div.